### DISCUSSION TITLE
2 How can I create chia plots with NFT in CLI

### DISCUSSION THREAD

**Discussion by heydari1** [Chia-Network/chia-blockchain#7365] [Q&A] [answered]
Hi
I tried to create NFT plot on power shell and I used following command : 

.\chia.exe plotnft create -s pool -k 32 -b 5000 -u 128 -r 2 -t E:\temp2 -d D:\05KH117 -n 10

but it didn't work, would you please guide me about correct command

**Comment by rmica** [comment, marked as answer]
"chia plotnft create" is to create a NFT
"chia plots create" is to create a plot (with -c option to set the NFT previously created)

**Reply by heydari1** [reply]
thank you so much
